.travis.yml

```diff
@@ -3,7 +3,7 @@
 language: go
 
 go:
-  - '1.11.x'
+  - '1.12.x'
```

Comment:
It might be worth to test the last 2 releases, so:
```
- '1.11.x'
- '1.x'
```

Reply:
Won't work in 1.11 though, as tls.VersionTLS13 is not defined there

Reply:
Very good point! That also means we should update the README to mention certigo requires Go 1.12 to compile.